hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2238,6 +2238,15 @@ public void deleteKey(OmKeyArgs args) throws IOException {
     }
   }
 
+  // TODO: HDDS-2424. recover-trash command server side handling.
+  @Override
+  public boolean recoverTrash(String volumeName, String bucketName,
+      String keyName, String destinationBucket) throws IOException {
+
+    boolean recoverOperation = true;
+    return recoverOperation;
```

Comment:
Even this is a partial work, is IT possible to add a unit test for it, which can be easily extended later for HDDS-2424.

Reply:
Thanks @xiaoyuyao for the review.

I updated the UT that is prototype for now,
and it can be extended for the following server-side development.